crates/astria-core/src/protocol/transaction/v1alpha1/action.rs

```diff
@@ -24,10 +24,15 @@ use crate::{
 };
 
 #[derive(Clone, Debug)]
+#[cfg_attr(feature = "serde", derive(::serde::Deserialize, ::serde::Serialize))]
+#[cfg_attr(
+    feature = "serde",
+    serde(into = "raw::Action", try_from = "raw::Action")
+)]
```

Comment:
These could probably be merged.  Same for `ValidatorUpdate` below.

Reply:
You are right, that does look neater. Done for both types in  17d857fc. There are many other instances in the code base where this could be done, but not in thits PR.